packages/webpack/README.md

```diff
@@ -10,6 +10,15 @@ We want to provide a good set of defaults on how to facilitate your web componen
 
 If you need support to use `import.meta.url` within webpack this is a minimal loader to support it.
 
+## Note
+This is NOT an optimal solution e.g. it will slow down your build time.
+However as currently `import.meta` results in an webpack parse error using a loader is probably the only thing we can do for now.
+For details see 
+- [https://github.com/webpack/webpack/issues/6719](https://github.com/webpack/webpack/issues/6719)
+- [https://github.com/Polymer/tools/issues/518](https://github.com/Polymer/tools/issues/518)
+
+Once the parse error is solved - it will probably work out of the box - if not we would advice to use [@babel/plugin-syntax-import-meta](https://babeljs.io/docs/en/next/babel-plugin-syntax-import-meta.html).
```

Comment:
The syntax plugin will not solve this. It only enables parsing so that it appears in the AST. You then need a transform plugin.

Reply:
replaced with
```
If webpack fixed that parse error import.meta will probably work out of the box.
If not then a babel plugin (that can work with AST) will be a better solution.
```
better?